### PR TITLE
Preliminary work to make local debugging of jekyll work.

### PR DESCRIPTION
**Pull request by byoshimi-github**
Running jekyll locally requires a Gemfile and null '---' header to convert .md files to .html (otherwise jekyll doesn't touch these files.)

Also added google_analytics to _config.yml.  Both _layout/*.md files appear to support this already.

### CODE REVIEW

**Comment by byoshimi-github** [comment]
Gemfile cleaned up.  .md references resolve correctly now.   Added `bundle install`.  You should be able to test this locally now using instructions in `debug_docs.md`

File: docs/developers/index.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+---
```
Comment (rchen152) [inline docs/developers/index.md:1]: Are the changes to this file and to index.md below necessary?

File: docs/Gemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "webrick"
+gem "github-pages", "~> 226", group: :jekyll_plugins
+
```
Comment (rchen152) [inline docs/Gemfile:6]: nit: Could you remove the extra newlines at the end of Gemfile, _config.yml, and debug_docs.md?

**Review by rchen152 — CHANGES_REQUESTED** [review]

**Comment by byoshimi-github** [comment]
Done.

**Review by rchen152 — APPROVED** [review]
Thanks, I'll merge once the checks finish running.

**Pull request closed without merging**